### ISSUE TITLE
Feature Request: Readme.md examples of usage

### ISSUE THREAD

**Issue by hack-r** [tidyverse/forcats#148]
I hate to complain, but it strikes me as very odd and unfortunate that the Read Me doesn't actually include any example usage or comments on `forcats` functions... it just talks about factors in general, to the extent you'd probably already be familiar with them when you're researching this library.

Explaining some concrete use cases and key functions of `forcats` would be a major improvement IMHO

**Comment by batpigandme** [comment]
@robinsones,
Would you still be interested in doing a PR for this or non-README forcats content? (I remember we had a discussion about this a while back, but I can't for the life of me find it on GitHub 😞).

**Comment by robinsones** [comment]
Yes, I actually was just working on it! It's getting a bit long so I wonder if a vignette is better ... but @batpigandme why don't I do a pull request with what I have and you can see?

**Comment by batpigandme** [comment]
Sounds good! No rush.
